app/assets/stylesheets/scheduler.css

```diff
@@ -194,9 +184,23 @@ body.modal-open {
 	100% {background-color: #15214B;}
 }
 
-.loading {
-	-webkit-animation-duration: 1s;
-	-webkit-animation-iteration-count: infinite;
-	-webkit-animation-name: colours;
-	-webkit-animation-timing-function: ease;
-}​
+#course-title {
+	text-align: center;
+}
+
+#scheduler-buttons > button {
```

Comment:
i would love you with all the love in the world if you can attach a screenshot here <3 <3 <3